app/views/users/show.html.erb

```diff
@@ -1,2 +1,14 @@
-<h1>This is a user</h1>
+<div class="users_container">
+<%= render partial: 'shared/card_user', locals: {user: @user} %>
+<p class="user_bio"><%= @user.bio%></p>
+<%#Add post for the user%>
+<% @posts.each do |post| %>
+<a href="<%=post.author_id%>/posts" class="user_link">
+<%= render partial: 'shared/card_post', locals: {post: post} %>
+</a>
+<%end%>
+<div class="btn"><button type="button" class="see_all_post">See all post</button></div>
```

Comment:
- [ ] As stated by the previous reviewer, kindly consider when you click on the see all posts take you to user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)

Reply:
@MeqdamAlqudah  this was never asked by the previous code reviewer.
Did you even Check the previous comment and check my code and see if ai made the changes?
The previous reviewer before you did not even look at my code and yet asked me to implement the changes requested by the previous reviewer here you are asking for the same changes that I have made